XenAdmin/Program.cs

```diff
@@ -225,7 +211,11 @@ static public void Main(string[] Args)
             AppDomain.CurrentDomain.UnhandledException += CurrentDomain_UnhandledException;
             Application.ThreadException -= Application_ThreadException;
             Application.ThreadException += Application_ThreadException;
+
+            // We must NEVER request ProfessionalColors before we have called Application.EnableVisualStyles
+            // as it may prevent the program from displayed as expected.

```

Comment:
Nit: `displayed` => `displaying`.

Reply:
I missed this! I'll correct it in a future PR.